sdk/nodejs/runtime/closure/codePaths.ts

```diff
@@ -46,6 +47,11 @@ export interface CodePathOptions {
      * used at runtime.
      */
     extraExcludePackages?: string[];
+
+    /**
+     * The resource to log any errors we encounter against.
+     */
+    logResource?: Resource;
 }
```

Comment:
When computing codepaths for a Lambda/FunctionApp it would be good to associate with a resource to more specifically narrow down where the problem is.